packages/manager/src/features/Databases/DatabaseDetail/DatabaseSummary/DatabaseSummaryConnectionDetails.tsx

```diff
@@ -143,22 +144,22 @@ export const DatabaseSummaryConnectionDetails: React.FC<Props> = (props) => {
   };
 
   const ssl = database.ssl_connection ? 'ENABLED' : 'DISABLED';
+  const disableShowBtn = ['provisioning', 'failed'].includes(database.status);
+  const disableToolTipText =
+    database.status === 'provisioning'
+      ? 'Your Database Cluster is currently provisioning.'
+      : 'You cannot view your root password when your Database Cluster has failed.';
```

Comment:
I wonder if it should be more like this because it lets the user know we're not just hiding it from them to be annoying as it a real technical limitation

```suggestion
      : 'Your root password is unavailable when your Database Cluster has failed.';
```

Reply:
I agree with this change